MediaRecorder.bs

```diff
@@ -241,6 +249,32 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Let |videoKeyFrameIntervalDuration| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalDuration]]=] slot if defined, and let
+    |videoKeyFrameIntervalCount| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalCount]]=] slot if defined. The UA SHOULD constrain the
+    configuration of |recorder| so that the video encoders follow the below rules:
+
+    <ol>
+    <li>If neither of the slots are defined, the User Agent may emit key frames as
+    it deems fit.</li>
```

Comment:
For consistency I think we should reference the let variables not "the slots". if it's unclear what "neither" refers to, I wonder if it might work to move it to the bottom (after "both")?

Reply:
Done.